plugins/woocommerce/includes/wc-update-functions.php

```diff
@@ -2667,3 +2667,10 @@ function wc_update_870_prevent_listing_of_transient_files_directory() {
 	$wp_filesystem->put_contents( $default_transient_files_dir . '/.htaccess', 'deny from all' );
 	$wp_filesystem->put_contents( $default_transient_files_dir . '/index.html', '' );
 }
+
+/**
+ * Add woocommerce_launch_your_store_tour_hidden.
+ */
+function wc_update_890_add_launch_your_store_tour_option() {
+	update_option( 'woocommerce_launch_your_store_tour_hidden', 'no' );
```

Comment:
I think we need to loop through all users and apply user meta here, instead of a site-wide option